src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceConfiguration.java

```diff
@@ -14,7 +14,6 @@
     private String applicationName;
     private long projectCreateTimeoutSeconds;
     private String projectId;
-    private String coreBillingAccount;
```

Comment:
Right now, we never do core project creation in the running system. I think everything is constructed at Terraform time. If that changes, we will need to know a billing account for those allocations somehow.
Not that we shouldn't make this change. Just that we may have to redo it down the line.

Reply:
So, I found out yesterday that we do create projects from the app (though it's busted because we need to manually enable firestore).  Do you mean something different by core project?  Are you referring to the main project that hosts the main app?  Yeah, we may eventually need billing info if we want to power billing monitoring but I agree that if we do that, we should explicitly re-add the property here as part of that feature work

Reply:
We call the project that the kubernetes and all of our parts runs in the core project.